x-pack/plugins/apm/server/index.ts

```diff
@@ -29,6 +29,7 @@ export const config = {
     ui: schema.object({
       enabled: schema.boolean({ defaultValue: true }),
       transactionGroupBucketSize: schema.number({ defaultValue: 1000 }),
+      maxServiceEnvironments: schema.number({ defaultValue: 100 }),
```

Comment:
I'm not sure we should have a "ui" section. Doesn't all the settings affect the ui somehow? Eg `serviceMapEnabled: false` will certainly be visible in the ui... Should it then be under `ui`? Makes it a little hard to draw the line about which things belong there or not.

Reply:
It's true that the nesting under `ui` is pretty arbitrary, we have similar configurations for bucket sizes and max items both with and with that `ui` section, i'll go ahead and change the new setting to remove it, but we'll have to wait for the 8.0 to migrate other configs out of `ui`.